lib/driver.js

```diff
@@ -59,14 +70,19 @@ export class ChromiumDriver extends BaseDriver {
     jsonwpRequiredCaps,
     w3cCapabilities,
   ) {
-    const [sessionId, caps] = /** @type {[string, ChromiumDriverCaps]} */(await super.createSession(jsonwpDesiredCapabilities, jsonwpRequiredCaps, w3cCapabilities));
-    await this.startChromedriverSession();
-    return [sessionId, caps];
+    const [sessionId,] = /** @type {[string, ChromiumDriverCaps]} */(await super.createSession(jsonwpDesiredCapabilities, jsonwpRequiredCaps, w3cCapabilities));
+    const returnedCaps = await this.startChromedriverSession();
+    if (returnedCaps.webSocketUrl) {
+      // @ts-ignore while webSocketUrl is a boolean when sent by the client, it's a string when it
```

Comment:
can we just cast it explicitly to string?

Reply:
you have to do this:

```js
this._bidiProxyUrl = (/** @type {string|null} */(/** @type {unknown} */(returnedCaps.webSocketUrl)));
```

not sure it's better 🤷